internal/cli/cli.go

```diff
@@ -122,3 +124,125 @@ func (opts *deleteOpts) FailMessage() string {
 	}
 	return fallbackFailMessage
 }
+
+type AtlasAlertsConfig struct {
```

Comment:
[nit] let's follow our naming convention for this structs
```suggestion
type atlasAlertsConfigOpts struct {
```
Lets please also add documentation on what to expect when using this